portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,225 @@
-#content {
+p {
+  font-size: 40px
+}
+#logo{
+  text-align: center;
+  background-image: url("https://media.giphy.com/media/4rwQehrSJTS6s/giphy.gif");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+#lll {
```

Comment:
Can you give some of these classes more meaningful names?  It's hard to tell what it refers to

Reply:
Resolving because I fixed class names